src/cli/commands/force/lightning/local/setup.ts

```diff
@@ -4,104 +4,13 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-import util from 'util';
 
-import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
-import { Logger, Messages, SfdxError } from '@salesforce/core';
+// tslint:disable: no-submodule-imports
 
-import { AndroidEnvironmentSetup } from '../../../../../common/AndroidEnvironmentSetup';
-import { CommandLineUtils } from '../../../../../common/Common';
-import { IOSEnvironmentSetup } from '../../../../../common/IOSEnvironmentSetup';
-import { LoggerSetup } from '../../../../../common/LoggerSetup';
-import {
-    BaseSetup,
-    Requirement,
-    SetupTestResult
-} from '../../../../../common/Requirements';
-
-// Initialize Messages with the current plugin directory
-Messages.importMessagesDirectory(__dirname);
-
-// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
-// or any library that is using the messages framework can also be loaded this way.
-const messages = Messages.loadMessages('@salesforce/lwc-dev-mobile', 'setup');
-
-export default class Setup extends SfdxCommand {
-    public static description = messages.getMessage('commandDescription');
-
-    public static readonly flagsConfig: FlagsConfig = {
-        platform: flags.string({
-            char: 'p',
-            description: messages.getMessage('platformFlagDescription'),
-            longDescription: messages.getMessage('platformFlagDescription'),
-            required: true
-        })
-    };
-
-    public examples = [
-        `sfdx force:lightning:local:setup -p iOS`,
-        `sfdx force:lightning:local:setup -p Android`
-    ];
-
-    private setupSteps: BaseSetup | undefined;
+import coreSetup from '@salesforce/lwc-dev-mobile-core/lib/cli/commands/force/lightning/local/setup';
```

Comment:
Is `coreSetup` something that has since changed? I'm not seeing this in the core codebase at this point.

Reply:
No it is not. I just wanted to use an alias for `Setup` that comes from `core` vs `Setup` command defined in this file (otherwise I would get compiler error due to name clash)

Reply:
Hmm. The strangeness is basically because we're declaring `Setup` in core as default:
```typescript
export default class Setup extends SfdxCommand {}
```
Which puts me in the same camp around `default` as [this guy](https://basarat.gitbook.io/typescript/main-1/defaultisbad). Because if we *didn't* declare it as `default`, and simply exported it, we could do:
```typescript
import { Setup as CoreSetup } from '@salesforce/lwc-dev-mobile-core/lib/cli/commands/force/lightning/local/setup';
```
Which IMO has more clarity and is a lot more deterministic in purpose.

Reply:
Very interesting points in that link. Ok I'm sold. I'll update `core` and then this here too.

Reply:
ok updated using [this commit](https://github.com/forcedotcom/lwc-dev-mobile-core/pull/3/commits/9802e3e197c54b019e95b2ff83ea0de7af55b832) of another PR in `core`. Once that goes it, I'll update this PR to (a) consume a newer `core` and (b) update the import.